### ISSUE TITLE
Placeholder colides with Chome Autofill

### ISSUE THREAD

**Issue by daslicht** [Dogfalo/materialize#6137]
Hi,
placeholders collide with Google Chomes autofill.
It looks like this
![](https://i.imgur.com/eV2vB7e.png)

Is there a way to keep autofill and preseve a good looking GUI ?

**Comment by gertjanekkel** [comment]
What version of MaterializeCSS do you use? I don't have this issue with the lastest version (1.0.0).

You could try something like this (using jQuery):


        setTimeout(function() {

            // Check values
            $('.ClassOnElement').each(function(){

                // Check value
                if ($(this).val().length > 0) $(this).closest('label').addClass('active');

            });

        },200);

**Comment by daslicht** [comment]
Ahh I see , I use this:
https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/css/materialize.min.css
Let me try the release

**Comment by daslicht** [comment]
same with teh release here:

![](https://i.imgur.com/FDzZzFt.png)

You are on macOS ? probably it is a mac thing

**Comment by gertjanekkel** [comment]
Both tested on Windows and MacOS (Chrome + Safari), no problems.

**Comment by daslicht** [comment]
That how it behaves like here:
![](https://media.giphy.com/media/4PY4bujNnSnwnO4qKH/giphy.gif)

**Comment by daslicht** [comment]
Here some more info:
Version 69.0.3497.100 (Official Build) (64-bit)
macOS Mojave (18A391)

**Comment by daslicht** [comment]
Hm , that is AutoComplete, not autofill ?

**Comment by daslicht** [comment]
Anyways , I simply dont use such kind of inputs until i find a solution which looks great anywhere

**Comment by sraboncmc** [comment]
Yes i agree with @daslicht . it is AutoComplete. It can be stopped by using autocomplete="off", Using "M.updateTextFields()" can be a solution.
